src/ir_Fujitsu.cpp

```diff
@@ -235,12 +240,17 @@ uint8_t IRFujitsuAC::getStateLength(void) {
   return updateUseLongOrShort() ? _state_length : _state_length_short;
 }
 
+/// Is the current binary state representation a long or a short code?
+/// @return true, if long; false, if short.
+bool IRFujitsuAC::isLongCode(void) const {
+  return _.Cmd == 0xFE || _.Cmd == 0xFC;
+}
+
 /// Get a PTR to the internal state/code for this protocol.
 /// @return PTR to a code for this protocol based on the current internal state.
 uint8_t* IRFujitsuAC::getRaw(void) {
   checkSum();
-  if (_.Cmd == 0xFE || _.Cmd == 0xFC)
-    return _.longcode;
+  if (isLongCode()) return _.longcode;
```

Comment:
Maybe return isLongCode() ? _.longcode : _.shortcode; ?

Reply:
I had thought of doing that. You've twisted my arm! ;-)

Done.